node/hybrid-node-tests/Common-test/Common-test.cjs

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
good job on those, but please correct the file & folder names - commonjs-test & ecmascript-test.